packages/firestore/src/lite/query.ts

```diff
@@ -795,7 +795,7 @@ function validateNewFilter(query: InternalQuery, filter: Filter): void {
       throw new FirestoreError(
         Code.INVALID_ARGUMENT,
         'Invalid query. All where filters with an inequality' +
-          ' (<, <=, >, or >=) must be on the same field. But you have' +
+          ' (<, <=, !=, >, or >=) must be on the same field. But you have' +
```

Comment:
I think this also needs to include `in` and `not-in`, which is rejected by the backend as well.

Reply:
Good catch! Added.